deploy/deploy-api.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Document the list of assumptions both for the user and for us if we make changes to our deployment stack that would potentially break this: cluster running on GKE named elasticsearch-cluster, kubectl and gcloud installed and configured globally, anything else?

Reply:
Example: I decide to rename `elasticsearch-cluster` to `es-cluster`, I update the data-explorer-indexers repo accordingly. Result is that this script breaks

Since the nature of this is that it will be fairly fragile against changes like that, documenting exactly what the assumptions are will be useful to prevent introducing bugs.

Reply:
Well, documenting here won't be useful to someone changing data-explorer-indexers repo.

Eventually we'll have CD. If someone makes a breaking change like that, we will get a CD failure email.

Reply:
If you're heavily against the documentation here I won't block you on it. I still think it would be useful to list assumptions to this script for the same reasons why we always document public methods with what kinds of inputs are acceptable.

Reply:
Done